server/perform_search.py

```diff
@@ -15,10 +15,12 @@ def get_similar_images(self, keyword, semantic_query, pixabay_max, top_k):
         :param top_k: Top-k images to return.
         :return: List of top-k PIL images.
         """
-        keyword_images = fetch_images_tag(keyword, pixabay_max)
+        keyword_images, keyword_image_urls = fetch_images_tag(keyword, pixabay_max)
```

Comment:
Images could be discarded then?

Reply:
Images still need to be there for being fed into the model?

Reply:
Correct. My bad.